data/laravel-framework.json

```diff
@@ -76,10 +76,40 @@
             "code": "php artisan migrate:reset"
           }, {
             "definition": "টেবিল গুলো মুছে ফেলে নতুন করে টেবিল তৈরি করতে চাইলে",
-            "code": "php artisan migrate:reset"
+            "code": "php artisan migrate:refresh"
```

Comment:
This is a bit unclear. 

> টেবিল গুলো মুছে ফেলে

By this you can mean two things:

1. **Drop** then **Migrate** execution.
2. **Rollback** all migrations then **Migrate** execution.

Notice these two mean different things. The first meaning that you are skipping the down function of all the migration files where as in the latter one you mean otherwise.

Reply:
Thank you, brother.